rtlib/src/Data.cpp

```diff
@@ -0,0 +1,74 @@
+//
+// Created by Anthony Tamasi on 2019-10-31.
+//
+
+#include <itkOpenCVImageBridge.h>
+
+#include "rt/Data.hpp"
+#include "rt/io/OBJReader.hpp"
+
+
+using namespace Data;
+
+// IO
+using OCVBridge = itk::OpenCVImageBridge;
+
+
+Data::Pointer Data::Load(const fs::path& path)
+{
+    //HOW TO GET METADATA AKA FILE TYPE? Could just read the 'metadata.json' file from the current directory right? Since each
```

Comment:
Yep.

Reply:
Do you have a specific library you would like the project to use for reading JSON metadata file? Or would boost::property_tree work? I know you'd like to phase boost out eventually so wanted to double check